### ISSUE TITLE
Feature Request: Download Overpass Query

### ISSUE THREAD

**Issue by stegers** [dw-innovation/kid2-spot#11]
It would be nice if you could download the query, too, in order to make manual changes.

**Comment by dendelyne** [comment]
Thanks for your feedback!

We are not using Overpass in the backend, but our own query graph representation and OSM query engine. 
Allowing direct access to the query representation may be possible in the future, although we are also exploring the possibility of allowing easier editing of queries via the GUI to make this feature more accessible.

**Comment by cutterkom** [comment]
Yes, supporting this. Just wanted to open the same issue. One thing is using spot to build a query, another is it to use it in scripts to get the data. also, because spot does not always yield the same results, I guess due to LLM
